grasa_event_locator/views.py

```diff
@@ -47,13 +157,44 @@ def editEvent(request):
 
 def event(request, eventID):
         event = Program.objects.get(pk=eventID)
+        grades_list_pub = ""
+        timing_list_pub = ""
+        gender_list_pub = ""
+        transportation_list_pub = ""
+        topic_list = event.categories.filter(id__lte=18)
+        grades_list = event.categories.filter(id__gte=21)
+        grades_list = grades_list.filter(id__lte=25)
+        for g in grades_list:
+                grades_list_pub = grades_list_pub + str(g) + ", "
+        grades_list_pub = grades_list_pub[:-2]
+
+        timing_list = event.categories.filter(id__gte=29)
+        timing_list = timing_list.filter(id__lte=34)
+        for t in timing_list:
+                timing_list_pub = timing_list_pub + str(t) + ", "
+        timing_list_pub = timing_list_pub[:-2]
+
+        gender_list = event.categories.filter(id__gte=26)
+        gender_list = gender_list.filter(id__lte=28)
+        for g in gender_list:
+                gender_list_pub = gender_list_pub + str(g) + ", "
+        gender_list_pub = gender_list_pub[:-2]
 
-        context = {'event' : event}
+        transportation_list = event.categories.filter(id__gte=19)
+        transportation_list = transportation_list.filter(id__lte=20)
+        for t in transportation_list:
+                transportation_list_pub = transportation_list_pub + str(t)
+        transportation_list_pub = transportation_list_pub[15:]
+
+        context = {'event' : event, 'topic_list' : topic_list, 'grades_list_pub' : grades_list_pub, 'timing_list_pub' : timing_list_pub, 'gender_list_pub' : gender_list_pub, 'transportation_list_pub' : transportation_list_pub}
```

Comment:
In a second pass, some comments in this method to explain what's going on will help for readability.

Reply:
Will comment this when I get a moment. But basically, the string containing "Transportation Provided" and "Transportation Not Provided" needed to have the work Transportation cut off, so that the event page does not indicate "Transportation: Transportation Provided", which felt redundant.